docs/manual/forms/examples/prefill.rst

```diff
@@ -1,12 +1,18 @@
-=======
-Prefill
-=======
-
-In dit voorbeeld tonen we hoe u een prefill kunnen gebruiken.  Prefill is een optie zodat
-een formulier prefilled met informatie is.  Met dit voorbeeld tonen we hoe u de
-:ref:`KvK <configuration_prefill_kvk>` prefill kan gebruiken maar we hebben op dit moment ook
-:ref:`StUF-BG <configuration_prefill_stuf_bg>` prefill en
-:ref:`Haal Centraal Bevragen personen <configuration_prefill_haal_centraal>` prefill dat u kan gebruiken.
+.. _example_prefill:
+
+=========================
+Voorinvullen van gegevens
+=========================
+
+In dit voorbeeld tonen we hoe u velden kunt voorinvullen. In dit voorbeeld 
```

Comment:
"In dit voorbeeld" start in both sentences is a bit weird

Reply:
Iets herschreven.